### DISCUSSION TITLE
Elastisearch is running but inidces are  not generated

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#314] [Installation] [answered]
Hi 

CSP:GCP

indices are not populated

**Comment by beepdot** [comment, marked as answer]
Please add more details and context to this thread. I am unable to understand what is the issue with this information.

**Reply by tsprasath** [reply]
@keshavprasadms issue with neo4j has started properly.

fix is started neo4j and then the indices are get populated in ES
